code/modules/unit_tests/holidays.dm

```diff
@@ -0,0 +1,35 @@
+// test Jewish holiday
+/datum/unit_test/hanukkah_2123/Run()
+	var/datum/holiday/hebrew/hanukkah/hanukkah = new
+	TEST_ASSERT_EQUAL(hanukkah.shouldCelebrate(13, DECEMBER, 2123, 2, MONDAY), TRUE, "December 13, 2123 was not Hannukkah.")
```

Comment:
For this and everything else, use `TEST_ASSERT` when checking if something is TRUE or FALSE.

For example--`TEST_ASSERT(hannukah.shouldCelebrate(...), "This wasn't Hannukah")` for checking TRUE, and `TEST_ASSERT(!holiday.shouldCelebrate(...), "Holiday was celebrated when it shouldn't have been")` for checking FALSE.